apps/sms/js/recipients.js

```diff
@@ -184,6 +202,8 @@
     data.set(this, list);
     events.set(this, { add: [], remove: [], modechange: [] });
     view.set(this, new Recipients.View(this, setup));
+
+    this.validator = RecipientValidator.for(this);
```

Comment:
Do we have suitable ```WeakMap``` for this private member? I see we many ```WeekMap```s in recipients.js :)